specs/plugin-extensibility.md

```diff
@@ -22,13 +22,28 @@ Keys and associated certificates used for signing artifacts using Notation could
 
 Notation will invoke plugins as executable, pass parameters using command line arguments, and use standard IO streams to pass request/response payloads. This mechanism is used as Go language (used to develop [Notation Go library](https://github.com/notaryproject/notation-go)) does not have a [in built support](https://github.com/golang/go/issues/19282) to load and execute plugins that works across OS platforms. Other mechanisms like gRPC require every plugin to be implemented as a service/daemon.
 
+### Package and distribute plugin file
+
+To ensure plugin installation is compatible with Notation, plugin publisher SHOULD follow these conventions to package and distribute plugin files:
+
+* `.zip`, `.tar.gz`, and single plugin executable file formats are supported by Notation. Plugin executable file MUST follow the naming convention `notation-{plugin-name}` 
+* Plugin publisher SHOULD distribute the plugin file with an `https` URL to enable install a plugin from a remote source, or provide an archive file to enable install a plugin from filesystem
```

Comment:
I think we don't need this one? Plugin publishers distribute the above mentioned release formats in their `release page`. Whether to install directly from URL or download then install from file system is decided by the `plugin user`.
```suggestion
```

Reply:
The intention of this sentence is to remind plugin publishers that Notation supports installing a plugin from an `https` URL or from a file in the filesystem, so they can distribute their plugin via these two methods. 

I agree with you that plugin installation is decided by users. But plugin vendors should be aware of these installation methods.

Reply:
In addition, some plugin vendors may not release their plugins publicly for some reasons.